### DISCUSSION TITLE
Can't use academic email

### DISCUSSION THREAD

**Discussion by mafineeek** [community/community#69116] [GitHub Education]
### Hi everyone,

I don't know what to do, because when I want to renew my Github Education Pack, I need to use my academic email, which i don't have access to (my school does not provide for students any mailing software). When i use my Student Card, I'm getting rejected. 

What can I do?

**Comment by mantasu** [comment]
Hey, unfortunately, you won't be able to access the student dev pack, at least not currently. I'm quoting an [answer](https://github.com/orgs/community/discussions/68909#discussioncomment-7159743) for another person who had the same issue:
> Hey Siddhant,
>
> As I stated earlier, [GitHub Education Terms](https://education.github.com/schools/terms) have been updated recently. And now, they aren't approving or onboarding anyone to GitHub education with any verification method, except edu email.
>
> They haven't updated the onboarding process yet. Till then whenever you reapply, your request will be automatically rejected.
>
> Kindly mark the answer as answered.
> Thank you!
